tests/helper/utils.py

```diff
@@ -72,6 +72,17 @@ def get_file_perm(client, fname):
     if not "cannot statx" in error:
         return int(output)
 
+def check_kernel_module_exists(client, module_name):
+    """Check if a kernel module exists in any subdirectory of /lib/modules, considering compression."""
+
+    for ext in ["ko", "ko.gz", "ko.xz", "ko.bz2", "ko.zst"]:
+        filename = f"{module_name}.{ext}"
+        command = f"find /lib/modules/ -type f -name {filename}"
```

Comment:
Before, it was explicitly looked into `/lib/modules/$(uname -r)`. Is it safe to become more generic here?
Since this runs in a running system, would 
```suggestion
        command = f"find /lib/modules/$(uname -r) -type f -name {filename}"
```
be an option?

Reply:
Ok, sometimes, reading the PR description is helpful. Still, what would be if we have multiple kernel packages in the system, one comes with the module, one does not?

Reply:
> Is it safe to become more generic here?
I asked myself the same question. Given our Gardener OS build targets, it is safe to assume that only install one kernel package is installed per image.

However, this might change for some reason in Gardener OS or a spin-off project in the future. 

I really want to have these checks available in the default "chroot" test environment. I will try to find a better solution that is future-proof.